tests/avro/mock_schema_registry_client.py

```diff
@@ -142,8 +142,8 @@ def get_id_for_schema(self, subject, avro_schema):
     def test_compatibility(self, subject, avro_schema, version='latest'):
         raise ClientError("not implemented")
 
-    def update_compatibility(self, level, subject=None):
+    def update_compatibility(self, level):
         raise ClientError("not implemented")
 
-    def get_compatibility(self, subject=None):
+    def get_compatibility(self):
```

Comment:
Along with the other comments, both these parameters should be maintained

Reply:
Agreed - reverting these